### PR TITLE
Updated the Be-Secure README.md

### PR DESCRIPTION
**Pull request by jobyko**

### CODE REVIEW

**Comment by jobyko** [comment]
@asa1997  Please review the pull request.

@sumodgeorge

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+  
+
+The vision of Be-Secure community is to be one stop for all users to contribute and consume security projects to address security needs of open source projects.
+
+# Be-Secure Projects / Utilities:
+   One of the first utilities to be created as part of Be-Secure project is BeSman.
+
+   •	BeSman – is a command-line utility to provision customized environments for each open source tech stack. These environments are known as BeSman environments. Please refer the following link (https://github.com/Be-Secure/BeSman) to know more about BeSman. It is built using bash and can be easily customized to provision new environments for an open source tech stack based on open source project requirements. Anyone can leverage BeSman utility to build customized environments for their projects.
+  
+For more information on Be-Secure, please visit our [site](https://jobyko.github.io/Be-Secure_web/).
```
Comment (asa1997) [inline README.md:32]: If the contents have been pulled into the parent repo, please use the GitHub page of the same.

**Review by asa1997 — APPROVED** [review]
@panickervinod @sumodgeorge 
@jobyko  I have suggested some changes. Please do take a look at it.
**LGTM**

File: README.md (hunk 1 above)
Comment (jobyko) [reply README.md:32]: @asa1997 
Made the suggested changes.

**Pull request merged**